docs/Templates.md

```diff
@@ -37,29 +36,23 @@ Additionally, the result is piped through the built-in Go template function `js`
         "name": "pusher.email"
       }
     ],
-    "trigger-rule":
-    {
-      "and":
-      [
+    "trigger-rule": {
+      "and": [
         {
-          "match":
-          {
+          "match": {
             "type": "payload-hmac-sha1",
-            "secret": "{{ getenv "XXXTEST_SECRET" | js }}",
-            "parameter":
-            {
+            "secret": "{{ getenv \"XXXTEST_SECRET\" | js }}",
```

Comment:
Escaping these quotes is not accurate here.  This is a Go template, not a JSON file.

Reply:
True, I did not read the template.md file properly when I created this PR. The missing `[` is the only actual issue.